src/codegen.cpp

```diff
@@ -6985,6 +6992,16 @@ jl_compile_result_t jl_emit_code(
         "functions compiled with custom codegen params must not be cached");
     JL_TRY {
         std::tie(m, decls) = emit_function(li, src, jlrettype, params);
+        if (dump_emitted_mi_name_stream != NULL) {
+            jl_printf(dump_emitted_mi_name_stream, "%s\t", decls.specFunctionObject.c_str());
+            // The quotes break CSV parsing if there are any internal quotes in the Type
+            // name (e.g.in Symbol("...")). The \t delineator should be enough.
+            // (And there can't be any tabs in it because the tabs are printed as "\t")
+            //jl_printf(dump_emitted_mi_name_stream, "\"");
```

Comment:
Perhaps drop this commented quote printing line and that two lines below in accord with the (great!) immediately preceding comment, and touch up the comment? :)

Reply:
Thanks. done!